app/models/resource_action_workflow.rb

```diff
@@ -55,7 +57,10 @@ def validate_dialog
   end
 
   def create_values
-    create_values_hash.tap { |value| value[:src_id] = @target.id }
+    create_values_hash.tap do |value|
+      value[:src_id] = @target.id
+      value[:request_options] = @request_options if @request_options.any?
```

Comment:
@bzwei how would some one set the request_options. I have the attr_accessor method used to set the request _options, which saves it in an instance variable (@request_options)?